deployment/cdk/opensearch-service-migration/lib/stack-composer.ts

```diff
@@ -53,7 +53,9 @@ export class StackComposer {
         const openAccessPolicyEnabled = getContextForType('openAccessPolicyEnabled', 'boolean')
         const availabilityZoneCount = getContextForType('availabilityZoneCount', 'number')
         const migrationAssistanceEnabled = getContextForType('migrationAssistanceEnabled', 'boolean')
-        const sourceCWLogStreamARN = getContextForType('sourceCWLogStreamARN', 'string')
+        const MSKARN = getContextForType('MSKARN', 'string')
+        const MSKBrokers = getContextForType('MSKBrokers', 'object')
+        const MSKTopic = getContextForType('MSKTopic', 'string')
```

Comment:
Any issue when providing these options and having `migrationAssistanceEnabled` as true and running `cdk synth "*"`?